src/modules/feature/folder/Folder.vue

```diff
@@ -70,19 +78,29 @@ const {
 
 const { createPageInformation } = useSharedBoardPageInformation();
 
-const { fetchFiles, upload, getFileRecordsByParentId, deleteFiles } =
+const { fetchFiles, upload, getFileRecordsByParentId, deleteFiles, rename } =
 	useFileStorageApi();
 
+const authModule: AuthModule = injectStrict(AUTH_MODULE_KEY);
+const userRoles = ref(authModule.getUserRoles);
+const isStudent = computed(() => {
+	return userRoles.value.includes("student");
+});
```

Comment:
```suggestion
const isStudent = computed(() => 
	authModule.getUserRoles().includes("student");
);
```
Macht das nicht das selbe? Erst ein `ref` zu erstellen, irritiert mich.